src/plothist/plotters.py

```diff
@@ -1040,6 +1041,8 @@ def plot_data_model_comparison(
         The label for the model uncertainties. Default is "Model stat. unc.".
     data_uncertainty_type : str, optional
         What kind of bin uncertainty to use for data_hist: "symmetrical" for the Poisson standard deviation derived from the variance stored in the histogram object, "asymmetrical" for asymmetrical uncertainties based on a Poisson confidence interval. Default is "asymmetrical".
+    plot_only : str, optional
```

Comment:
I would put this argument after fig, ax_main, ax_comparison, so that these are defined and the explanation can be understood. We could also add that this argument can only be used when fig, ax_main and ax_comparison are not provided by the user.